### DISCUSSION TITLE
× Unhandled Rejection (DoesNotExist): getFile file.json failed.

### DISCUSSION THREAD

**Discussion by ladjalkhalil** [stx-labs/stacks.js#1069] [Q&A]
Hello 
i have a problem i wish i can get any help here
i making an app on stacks with react everything works fine until i try to change the name of the file in functions (putFile) and (getFile) putFile works fien but getFiles tells me that the file does not exists or it could not be found . .. this is the function i'm using to get file and store it's content in state array: 
async fetchData() {
  let fileName = 'message.json'
  const options = { decrypt: true };
  storage.getFile(fileName, options);
  let messages = JSON.parse(fileName || "[]");
  this.setState({
    messages,
    user: new Person(this.props.userSession.loadUserData().profile)
  });
}

**Comment by janniks** [comment]
Hi — Is this still an active problem?
